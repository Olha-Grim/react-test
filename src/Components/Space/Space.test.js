import React from "react";
import { render } from "@testing-library/react";
import Space from ".";

describe("Space", () => {
  test("interaction with Space component", () => {
    const { container } = render(<Space>Space</Space>);
    const space = container.firstChild;
    expect(space).toBeInTheDocument();
  });
});
