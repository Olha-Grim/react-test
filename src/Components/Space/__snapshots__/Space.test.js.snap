// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton interaction with Skeleton component 1`] = `<DocumentFragment />`;
